package.json

```diff
@@ -7,6 +7,14 @@
     "type": "git",
     "url": "https://github.com/ParsePlatform/parse-server"
   },
+  "files": [
+    "bin/",
+    "lib/",
+    "src/",
```

Comment:
Not sure why would we want `src/` here, since it's non-compiled version of `lib`.